### DISCUSSION TITLE
Not able to access devserver

### DISCUSSION THREAD

**Discussion by haldaranup** [learningequality/kolibri#11266] [Q&A] [answered]
My server has started on localhost:3000 but I am getting cannot get / error. I am not able to find what is the error as everything seems OK to me. Please help on this, attaching the SS for the same

![image](https://github.com/learningequality/kolibri/assets/76952588/d6103dfa-a50e-49af-b266-196bbfccf2d4)

**Comment by bjester** [comment, marked as answer]
Hi @haldaranup,

The webpack dev server runs on port 3000 to serve JS and CSS resources to the browser as you make changes to their source files while developing. The Kolibri app itself runs on port 8000 by default when you use the yarn scripts to run the development server. Did you try to access http://localhost:8000 ?

**Reply by haldaranup** [reply]
Ohh... I missed it, doc clearly mentioned to access the server on port 8000. Thank you so much @bjester its working now
